app/src/org/commcare/pn/helper/NotificationBroadcastHelper.kt

```diff
@@ -0,0 +1,43 @@
+package org.commcare.pn.helper
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.content.IntentFilter
+import androidx.lifecycle.DefaultLifecycleObserver
+import androidx.lifecycle.LifecycleOwner
+import androidx.localbroadcastmanager.content.LocalBroadcastManager
+object NotificationBroadcastHelper {
+    const val ACTION_NEW_NOTIFICATIONS = "com.dimagi.NEW_NOTIFICATIONS_RECEIVED"
```

Comment:
can we namespace this to the package instead `org.commcare.dalvik.action.NEW_NOTIFICATION`